src/sentry/api/endpoints/organization_traces.py

```diff
@@ -147,19 +136,18 @@ def data_fn(offset: int, limit: int):
                 project_slugs=None,
                 include_all_accessible=True,
             )
-            snuba_params.projects = all_projects
-            params["projects"] = snuba_params.projects
-            params["projects_objects"] = snuba_params.projects
-            params["projects_id"] = snuba_params.project_ids
-
-            trace_condition = f"trace:[{', '.join(spans_by_trace.keys())}]"
+            all_projects_snuba_params = dataclasses.replace(snuba_params, projects=all_projects)
+            all_projects_params = dict(params)
+            all_projects_params["projects"] = all_projects_snuba_params.projects
+            all_projects_params["projects_objects"] = all_projects_snuba_params.projects
+            all_projects_params["projects_id"] = all_projects_snuba_params.project_ids
 
             with handle_query_errors():
                 breakdowns_query = SpansIndexedQueryBuilder(
                     Dataset.SpansIndexed,
-                    cast(ParamsType, params),
-                    snuba_params=snuba_params,
-                    query=f"{trace_condition}",
+                    cast(ParamsType, all_projects_params),
+                    snuba_params=all_projects_snuba_params,
+                    query=None,
```

Comment:
w/o query how are you limited the breakdowns to the traces above

Reply:
Can follow up w/ this in another PR, lets get these changes in

Reply:
from slack will refactor so this condition is more obvious to read in a followup